### PR TITLE
Don't Load Legacy "Event Registration Options" or "Author" Meta Boxes in EDTR

### PR DESCRIPTION
**Pull request by tn3rb**
This PR simply skips loading the "Event Registration Options" and "Author" meta boxes when the advanced editor option is selected. It also avoids loading some other resources that are only relevant to the legacy dates and tickets section of the event editor.

### CODE REVIEW

File: admin_pages/events/Events_Admin_Page.core.php [hunk 1]
```diff
@@ -1569,16 +1569,25 @@ public function get_event_object()
...
+        if (EE_Registry::instance()->CFG->admin->useAdvancedEditor()) {
+            add_action(
+                'add_meta_boxes_espresso_events',
+                function () {
+                    global $current_screen;
+                    remove_meta_box( 'authordiv' , $current_screen , 'normal' );
+                },
+                99
             );
+            return;
```
Comment (manzoorwanijk) [inline admin_pages/events/Events_Admin_Page.core.php:1581]: 👍

File: caffeinated/admin/extend/events/Extend_Events_Admin_Page.core.php [hunk 2]
```diff
@@ -146,17 +107,60 @@ protected function _extend_page_config()
...
-        // partial route/config override
-        $this->_page_config['import_events']['metaboxes'] = $this->_default_espresso_metaboxes;
-        $this->_page_config['create_new']['metaboxes'][] = '_premium_event_editor_meta_boxes';
-        // don't load qTips if using the advanced editor
+        );        // don't load these meta boxes if using the advanced editor
         if (! $this->admin_config->useAdvancedEditor()) {
+            $this->_page_config['create_new']['metaboxes'][] = '_premium_event_editor_meta_boxes';
+            $this->_page_config['edit']['metaboxes'][] = '_premium_event_editor_meta_boxes';
             $this->_page_config['create_new']['qtips'][] = 'EE_Event_Editor_Tips';
             $this->_page_config['edit']['qtips'][] = 'EE_Event_Editor_Tips';
```
Comment (manzoorwanijk) [inline caffeinated/admin/extend/events/Extend_Events_Admin_Page.core.php:115]: I hope the JS and CSS assets related to these metaboxes are not loaded.

**Review by manzoorwanijk — APPROVED** [review]
Looks great! Finally feeling like we are replacing the whole editor with React 😄

File: caffeinated/admin/extend/events/Extend_Events_Admin_Page.core.php (hunk 2 above)
Comment (tn3rb) [reply caffeinated/admin/extend/events/Extend_Events_Admin_Page.core.php:115]: thnx... double checked and found a few that were still being loaded 👍

File: admin_pages/events/Events_Admin_Page.core.php [hunk 3]
```diff
@@ -562,19 +562,22 @@ public function load_scripts_styles()
...
-        // registers for all views
         // scripts
         wp_register_script(
             'event_editor_js',
             EVENTS_ASSETS_URL . 'event_editor.js',
-            array('ee_admin_js', 'jquery-ui-slider', 'jquery-ui-timepicker-addon'),
+            ['ee_admin_js', 'jquery-ui-slider', 'jquery-ui-timepicker-addon'],
             EVENT_ESPRESSO_VERSION,
             true
         );
```
Comment (manzoorwanijk) [inline admin_pages/events/Events_Admin_Page.core.php:583]: 👍

**Review by manzoorwanijk — APPROVED** [review]
Great

**Review by manzoorwanijk — APPROVED** [review]

**Pull request merged**